SolutionInfo.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Reflection;
 
-[assembly: AssemblyVersion("1.3.0.0")]
+[assembly: AssemblyVersion("2.0.0.0")]
```

Comment:
Why is a major change needed?

Reply:
New constructor parameters and making `position` required for `ServiceDefinitionException`.

Reply:
So, if we introduce constructor overloads (instead of the default parameter) and allow position to be null (even if we never use it that way) we can put off the major version change? I'd like to avoid bumping it until there's a breaking change to `Facility.Core` if at all possible (not to mention generally speaking).